src/content/docs/reference/Macros/MC_HAND_DEPTH.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: MC_HAND_DEPTH
+description: MC_HAND_DEPTH
+sidebar:
+  label: MC_HAND_DEPTH
+  order: 1
+---
+
+### `MC_HAND_DEPTH`
+
+The clip space depth multiplier applied to geometry in [`gbuffers_hand`](/reference/programs/gbuffers) and [`gbuffers_hand_water`](/reference/programs/gbuffers).
+
+The multiplier is applied in clip space automatically at the end of the program. When working backwards from the depth buffer, the multiplier should be applied in NDC space rather than screen space to produce the correct result, for example:
```

Comment:
```suggestion
The multiplier is applied in clip space automatically through the [projection matrix](/reference/uniforms/matrices/projectionMatrix]. When working backwards from the depth buffer, the multiplier should be applied in NDC space rather than screen space to produce the correct result, for example:
```